crates/ruff_server/src/session/settings.rs

```diff
@@ -469,6 +641,15 @@ mod tests {
                 lint_enable: true,
                 disable_rule_comment_enable: false,
                 fix_violation_enable: true,
+                editor_settings: ResolvedEditorSettings {
+                    lint_preview: None,
+                    format_preview: None,
+                    select: None,
+                    extend_select: None,
+                    ignore: Some(vec![RuleSelector::from_str("RUF001").unwrap()]),
+                    exclude: Some(vec![PathBuf::from_str("third_party").unwrap()]),
+                    line_length: Some(LineLength::from_u16_for_testing_only(80u16))
```

Comment:
Yeah, like the function is used exactly once. This doesn't warrent adding the method. I recommend using `LineLength::try_from(80).unwrap()`